audit/jsonformatter.go

```diff
@@ -0,0 +1,27 @@
+package audit
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	. "github.com/sirupsen/logrus"
+)
+
+type JsonFormatter struct {
+	PrettyPrint bool
+}
+
+func (f *JsonFormatter) Format(entry *Entry) ([]byte, error) {
+	// Note this doesn't include Time, Level and Message which are available on
+	// the Entry. Consult `godoc` on information about those fields or read the
+	// source of the official loggers.
+	var b *bytes.Buffer = &bytes.Buffer{}
```

Comment:
Pretty sure you can skip all the `bytes.Buffer` stuff and just use `serialized, err := json.Marshal(entry.Data)`, it returns a `[]byte, err`. You could just use https://golang.org/pkg/encoding/json/#MarshalIndent as well, rather than setting up the encoder, etc...

Reply:
Good call.... that cleans things up quite a bit. See here a52348a